registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DocumentScanController.java

```diff
@@ -407,7 +419,42 @@ private void initialize() {
 				documentVBox.getStyleClass().add(RegistrationConstants.SCAN_VBOX);
 				documentVBox.setId(documentCategory.getId());
 
-				documentVBoxes.put(documentCategory.getId(), documentVBox);
+				documentVBoxes.put(documentCategory.getId(), documentVBox);				
+				
+				VBox refNumVBox = new VBox();
+				refNumVBox.setId(docCategoryCode + "RefNumVBox");
+				refNumVBox.getStyleClass().add(RegistrationConstants.SCAN_VBOX);
+				TextField numberOfDocs = new TextField();
+				numberOfDocs.setId(docCategoryCode + "RefNum");
+				numberOfDocs.setPromptText("Ref Number");
+				numberOfDocs.getStyleClass().add(RegistrationConstants.DEMOGRAPHIC_TEXTFIELD);
+				numberOfDocs.setStyle("-fx-font-size:13");

```

Comment:
@HimajaDhanyamraju2  Can you pls add a separate styleclass and move the fontsize styling under the style class

Reply:
Modified as suggested